docs/user-guide/diffing.md

```diff
@@ -43,6 +43,16 @@ spec:
     - /spec/replicas
 ```
 
+To ignore elements of a list, you can use JQ path expressions to identify list items based on item content:
+```yaml
+spec:
+  ignoreDifferences:
+  - group: apps
+    kind: Deployment
+    jqPathExpressions:
+    - .spec.template.spec.initContainers[] | select(.name == "injected-init-container")
```

Comment:
Great example. I did not realize jq expressions are that powerful and supports this kind of logic